src/pulse3D/excel_writer.py

```diff
@@ -370,7 +372,10 @@ def write_xlsx(
         # window, interpolate, normalize, and scale data
         windowed_timepoints_us = interpolated_timepoints_us[start_idx:end_idx]
         interpolated_force = interp_data_fn(windowed_timepoints_us)
-        interpolated_force = (interpolated_force - min(interpolated_force)) * MICRO_TO_BASE_CONVERSION
+        if "xlsx" in plate_recording.path:
```

Comment:
It would be better to set an `is_optical_file` attribute on the plate_recording object and check that instead